apps/ledger-live-desktop/src/renderer/screens/platform/Catalog.jsx

```diff
@@ -53,7 +53,7 @@ const PlatformCatalog = () => {
   const history = useHistory();
 
   const { state } = useRemoteLiveAppContext();
-  const manifests = useMemo(() => (state.value ? state.value.liveAppByIndex : []), [state.value]);
+  const manifests = useMemo(() => (state.value ? state.value.liveAppByIndex : []), [state.value]).filter(manifest => !manifest.private);
```

Comment:
Why not put the `filter` inside the `useMemo`?

```suggestion
  const manifests = useMemo(
    () => (state.value ? state.value.liveAppByIndex.filter(manifest => !manifest.private) : []),
    [state.value],
  );
```